talk/owt/docker/android/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This Dockerfile creates a docker image for building the libwebrtc part of OWT
+# Android SDK.
+
+FROM ubuntu:20.04
+SHELL ["/bin/bash", "-c"]
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y git curl wget lsb-release tzdata xz-utils python
+RUN git config --global user.email "example@example.com"
+RUN git config --global user.name "Example Name"
+RUN mkdir workspace
+WORKDIR /workspace
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+ENV PATH="$PATH:/workspace/depot_tools"
+RUN mkdir owt-android
+WORKDIR /workspace/owt-android
+RUN echo $'solutions = [\n\
+  {\n\
+    "name"        : "src",\n\
+    "url"         : "https://github.com/open-webrtc-toolkit/owt-client-native.git",\n\
+    "deps_file"   : "DEPS",\n\
+    "managed"     : False,\n\
+    "custom_deps" : {\n\
+    },\n\
+    "custom_vars": {},\n\
+  },\n\
+]\n\
+target_os = ["android"]' > .gclient
+RUN gclient sync --nohooks
+RUN apt install 
```

Comment:
apt install ...., or apt update?

Another thing, please combine multiple RUN commands with:
RUN gclient sync --nohooks && \
        apt install abc && \

Reply:
Removed `apt install`. Also combined sync and runhooks. They were put in two separate commands to debug runhooks.

Reply:
my comment about combining multiple RUN is not specific to this gclient... in docker file every "RUN" command will create one layer, so we need to avoid them as many as possible.

Reply:
Merged some other commands. Not combine all commands because some command pulls a lot of data, put them in separate layers saves network traffic and time if one of them fails.